sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1543,6 +1543,70 @@ abstract class DynamicPartitionPruningSuiteBase
       checkAnswer(df, Row(1150, 1) :: Row(1130, 4) :: Row(1140, 4) :: Nil)
     }
   }
+
+  test("No dynamic partition pruning filters in case of static partition filtering") {
```

Comment:
Could you add `SPARK-36819:` prefix to the test case name?
```
- test("No dynamic partition pruning filters in case of static partition filtering") {
+ test("SPARK-36819: No dynamic partition pruning filters in case of static partition filtering") {
```

Reply:
added, thanks